### PR TITLE
add plotly tables

### PR DESCRIPTION
**Pull request by bvenn**
add table chart
add table documentation
add consolas font
update docu sidebar

### CODE REVIEW

File: src/FSharp.Plotly/FSharp.Plotly.fsproj [hunk 1]
```diff
@@ -25,6 +25,9 @@
     <Compile Include="Line.fs" />
     <Compile Include="Marker.fs" />
     <Compile Include="Font.fs" />
+    <Compile Include="Cellcolor.fs" />
+    <Compile Include="Header.fs" />
```
Comment (kMutagene) [inline src/FSharp.Plotly/FSharp.Plotly.fsproj]: Please merge these 2 files

File: src/FSharp.Plotly/Cellcolor.fs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿namespace FSharp.Plotly
```
Comment (kMutagene) [inline src/FSharp.Plotly/Cellcolor.fs]: Please create a type in StyleParam.fs for this

Comment (bvenn) [reply src/FSharp.Plotly/Cellcolor.fs]: Must be a dynamic object. Moved to Table.fs

**Comment by bvenn** [comment]
New commits resolve issue #27

**Pull request merged**